sr/comp/comp.py

```diff
@@ -36,6 +37,46 @@ def load_scorer(root: Path) -> ScorerType:
     return cast(ScorerType, score['Scorer'])
 
 
+def load_ranker(root: Path) -> RankerType:
+    """
+    Load the ranker module from Compstate repo.
+
+    :param Path root: The path to the compstate repo.
+    """
```

Comment:
Should we have a separated loading mechanism? I imagine there's quite a bit in common between loading a ranker and scorer, and possibly we might want to add more of these in the future. Might be good to factor out the `sys.path` alterations etc.

Reply:
Perhaps, I considered doing this but opted not to for speed for now. (We also have the same code again in srcomp-scorer, which would be great to further deduplicate). I've recorded an issue (https://github.com/PeterJCLaw/srcomp/issues/41) and will pick this up later.